SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -188,7 +196,7 @@
         EmaxFTFP    = cms.double(25.), # in GeV
         EmaxBERTpi  = cms.double(12.), # in GeV
         LowEnergyGflashEcal = cms.bool(False),
-        LowEnergyGflashEcalEmax = cms.double(100),
+        LowEnergyGflashEcalEmax = cms.double(0.02), # in GeV
```

Comment:
@civanch could you confirm that you changed this value on purpose?

Reply:
@silviodonato , yes, I confirm. Thanks for merging of this PR.